### ISSUE TITLE
Usage requires esModuleInterop, or results in compilation error

### ISSUE THREAD

**Issue by rosslavery** [Opteo/google-ads-node#21]
### I'm submitting a ...
  - [x] bug report
  - [ ] feature request
  - [ ] question about the decisions made in the repository
  - [ ] question about how to use this project

### Summary
Simple TypeScript installation of the latest `google-ads-api` fails to compile due to the following error:
```
node_modules/google-ads-node/build/lib/utils.d.ts(1,8): error TS1192: Module '"/usr/app/node_modules/@types/google-protobuf/google/protobuf/field_mask_pb"' has no default export.
```

`tsconfig.json` in this repo has `esModuleInterop: true`, which is a non-default compiler option.

Projects that don't have this enabled are unable to use `google-ads-api` and `google-ads-node` as a result.

### Other information
Enabling `esModuleInterop` on my project fixes the above error but created many more errors (unrelated to `google-ads-node` as a result.

![Screen Shot 2019-08-23 at 3 00 37 PM](https://user-images.githubusercontent.com/656884/63617231-e6e50800-c5b6-11e9-8ffd-d4011d8f7061.png)

**Comment by avermeil** [comment]
Hi @rosslavery -- thanks for the bug report :)

We've just released a new version, `v3.2.1`, which should solve this issue.

Let us know if it works for you!

**Comment by rosslavery** [comment]
All good here, thank you!
